code/backend/batch/AddURLEmbeddings.py

```diff
@@ -17,34 +22,54 @@ def add_url_embeddings(req: func.HttpRequest) -> func.HttpResponse:
     logger.info("Python HTTP trigger function processed a request.")
 
     # Get Url from request
-    url = req.params.get("url")
-    if not url:
-        try:
-            req_body = req.get_json()
-        except ValueError:
-            pass
-        else:
-            url = req_body.get("url")
```

Comment:
I removed the above conditional logic since the one place calling this API sends the URL in the body